README.md

```diff
@@ -1,37 +1,70 @@
 
 Demonstrated in Cloudlab, using 3 physical machines each running Ubuntu 16.04. The topology information can be found here: https://www.cloudlab.us/manage_profile.php?action=edit&uuid=0d1e3689-b5bb-11e7-b179-90e2ba22fee4
 
 Topology picture:   Device 1 -- Device 2 -- Device 3
+![alt text](https://i.ibb.co/JpkNXzv/image.png)
 
 - Device 1: "Node_0" emulates the user/attacker trying to access the IoT device (IP address: 192.1.1.2)
-- Device 2: "Dataplane" emulates the software-defined security gateway (both the controller and dataplane are on this host). It is running OVS to create a virtual switch for routing IP traffic through the middlebox specified in the policy (policy/newPolicy0.json). It runs a very basic "controller" (simple-controller.py) to dynamically change the middlebox and routing of IP traffic based upon events it receives from the middlebox.
+- Device 2: "Dataplane" emulates the software-defined security gateway (both the controller and dataplane are on this host). It is running OVS to create a virtual switch for routing IP traffic through the middlebox specified in the [policy](https://github.com/brytul/IoT_Sec_Gateway/blob/master/policies/cloudlab-NewPolicy20.json). It runs a very basic "controller" (simple-controller.py) to dynamically change the middlebox and routing of IP traffic based upon events it receives from the middlebox.
 - Device 3: "Node_1" emulates an IoT device (IP address: 10.1.1.2)
 
 
-Running the experiment:
-1. Initial device setup:
-
---A) Run setup-node.sh on the systems emulating the user/attacker and the IoT device (Device 1 & Device 3).
-
----- Runs apt-get update, docker, (modified version of) ovs (includes ovs-docker-remote) and ensures that the ip routing is setup to use the proper NIC for transmitting packets through the software-defined security gateway.
-
-  ------ Hosts may require removing routes that are specified 'via' the dataplane host. 
-
---B) Run setup-data-plane.sh on the system emulating the software-defined security gateway (Device 2).
-
----- Runs apt-get update, docker, & (modified version of) OVS (includes ovs-docker-remote). Builds snort containers for this emonstration (docker_containers/*). Disables GRO, and sets up a bridge (br0) between the NIC connected to Device 1 (enp6s0f0) and the NIC connected to Device 3 (enps6s0f1).
+## Steps for running the experiment in CloudLab
 
----- NOTE: at this point, Device 1 and Device 3 should be able to send packets (i.e. ping) between each other. However, the IP addresses associated with Device 2's NICs will no longer be usable.
+  - **1) Initial setup**
+      - Run the following command on all 3 nodes:
+      `git clone https://github.com/slab14/IoT_Sec_Gateway.git`
+      
+  - **2) Setup each node**
+  
+      - On "Node_0" and "Node_1", `cd` into __IoT_Sec_Gateway__ and run the following:
+      `./setup-node.sh`
+      - On "Dataplane", `cd` into __IoT_Sec_Gateway__ and run the following:
+      `./setup-data-plane.sh`
```

Comment:
Can we add a few words here, stating some of the configuration items that this script is performing (i.e. cloning OVS, ODL, setting up remote listening interfaces, etc)?